packages/api/src/external/commonwell/document/document-query.ts

```diff
@@ -93,20 +98,26 @@ export async function queryAndProcessDocuments({
   ignoreFhirConversionAndUpsert?: boolean;
   requestId: string;
 }): Promise<void> {
-  const { log } = Util.out(`CW queryDocuments: ${requestId} - M patient ${patientParam.id}`);
+  const { id: patientId, cxId } = patientParam;
+  const { log } = Util.out(`CW queryDocuments: ${requestId} - M patient ${patientId}`);
+
+  const interrupt = buildInterrupt({ patientId, cxId, log });
+  if (!(await isCWDirectEnabledForCx(cxId))) {
+    return interrupt(`CW Direct is not enabled for cx ${cxId}`);
```

Comment:
nit: why not use the existing format we use for CQ? i.e. 
`CQ disabled for cx ${cxId}`